spec/models/state_file_nc_intake_spec.rb

```diff
@@ -76,9 +76,14 @@
 require 'rails_helper'
 
 RSpec.describe StateFileNcIntake, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it_behaves_like :state_file_base_intake, factory: :state_file_nc_intake
 
   describe "#calculate_sales_use_tax" do
-    # TODO: Add tests in FYST-426
+    let(:intake) { create :state_file_nc_intake }
+
+    it "calculates the sales use tax using the nc_taxable_income" do
```

Comment:
this is calculation is explored more on line 127 of `spec/lib/efile/nc/d400_calculator_spec.rb`